metron-streaming/Metron-Common/src/test/java/org/apache/metron/pcap/PcapUtilsTest.java

```diff
@@ -6,17 +6,26 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
  * Unless required by applicable law or agreed to in writing, software
```

Comment:
I'm a little surprised that checkstyle didn't object to this. I think this was added by a format code call in your dev IDE.